src/Symfony/Component/HttpFoundation/ResponseStatusCode.php

```diff
@@ -0,0 +1,282 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation;
+
+use Teapot\StatusCode\RFC\RFC2295;
+use Teapot\StatusCode\RFC\RFC2324;
+use Teapot\StatusCode\RFC\RFC2518;
+use Teapot\StatusCode\RFC\RFC2774;
+use Teapot\StatusCode\RFC\RFC2817;
+use Teapot\StatusCode\RFC\RFC3229;
+use Teapot\StatusCode\RFC\RFC3648;
+use Teapot\StatusCode\RFC\RFC4918;
+use Teapot\StatusCode\RFC\RFC5842;
+use Teapot\StatusCode\RFC\RFC6585;
+use Teapot\StatusCode\RFC\RFC7231;
+use Teapot\StatusCode\RFC\RFC7232;
+use Teapot\StatusCode\RFC\RFC7233;
+use Teapot\StatusCode\RFC\RFC7235;
+use Teapot\StatusCode\RFC\RFC7538;
+use Teapot\StatusCode\RFC\RFC7540;
+use Teapot\StatusCode\RFC\RFC7725;
```

Comment:
which breaking change ? Changing the value of any of these constants would be a breaking change anyway, as it would make the code return a different status code by updating Symfony. Values of these constants are **not** internal things we can change over time.